release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -197,6 +197,15 @@ For more information, see xref:../networking/external_dns_operator/nw-installing
 
 SR-IOV support is now available for xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Pensando DSC cards]. OpenShift SR-IOV is supported, but you must set a static, Virtual Function (VF) media access control (MAC) address using the SR-IOV CNI config file when using SR-IOV.
 
+[id="ocp-4-11-support-for-hosted-control-planes"]
+==== Hosted control planes (Technology preview)
+
+A technology preview is available in {product-title} version 4.11 for hosted control planes. This new deployment model removes the requirement that clusters be self-hosted. Instead, a hosted cluster's control plane exists as pods on another management cluster.
```

Comment:
we should mention that it's available when installing the multicluster engine for kubernetes, and references the docs that laura wrote if possible https://github.com/openshift/openshift-docs/pull/47565#pullrequestreview-1036483127 

```suggestion
This new deployment model removes the requirement that clusters be self-hosted. Instead, a hosted cluster's control plane exists as pods on another management cluster.
```
Actually, clusters can still be self-hosted, i.e., self-hosted control-plane planes. Also, the "instead" part of the sentence above is not an alternative to the first clause of the sentence. 

We could say something like. 

This deployment model enables hosting OpenShift clusters at scale with reduced infrastructure costs, operational overhead, and hard separation of concerns between management and workloads. For more information, check https://github.com/openshift/openshift-docs/pull/47565#pullrequestreview-1036483127